pkgs/development/python-modules/import-expression/default.nix

```diff
@@ -0,0 +1,45 @@
+{ lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  pytest,
+  pytest-cov,
+  astunparse,
+}:
+buildPythonPackage rec {
+  pname = "import-expression";
+  version = "1.1.4";
+  src = fetchFromGitHub {
+    owner = "ioistired";
+    repo = "import-expression-parser";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-mll2NePB7fthzltLOk6D9BgaDpH6GaW4psqcGun/0qM=";
+  };
+  pythonImportsCheck = [
+    "import_expression"
+    "import_expression._codec"
+  ];
+  propagatedBuildInputs = [ astunparse ];
+  nativeCheckInputs = [ pytest pytest-cov ];
+  pytestFlagsArray = [
+    "tests.py"
+    "--cov import_expression"
+    "--cov-config .coveragerc"
+    "--cov-report term-missing"
+    "--cov-report html"
+    "--cov-branch"
+  ];
```

Comment:
```suggestion
```
The reason we test is to make sure the package works as expected.
So there is no need to measure coverage.